src/Allegro.DotnetSdk/version.json

```diff
@@ -1,4 +1,4 @@
 {
   "inherit": true,
-  "version": "1.0.2"
+  "version": "1.0.3"
```

Comment:
Looks like version is also specified in the csproj file - it should probably be removed from there to avoid confusion:
https://github.com/allegro/dotnet-sdk/blob/35080f056467ca923109aa3b83fbc78bad5cbe4a/src/Allegro.DotnetSdk/Allegro.DotnetSdk.csproj#L10